docs/ingestion/native-batch.md

```diff
@@ -294,11 +294,17 @@ How the worker task creates segments is:
 |property|description|default|required?|
 |--------|-----------|-------|---------|
 |type|This should always be `hashed`|none|yes|
-|numShards|Directly specify the number of shards to create. If this is specified and `intervals` is specified in the `granularitySpec`, the index task can skip the determine intervals/partitions pass through the data.|null|yes|
+|numShards|Directly specify the number of shards to create. If this is specified and `intervals` is specified in the `granularitySpec`, the index task can skip the determine intervals/partitions pass through the data.|null|no|
```

Comment:
I don't think I added any new parameters to `HashedPartitionsSpec`, were you referring to something else?

Reply:
Oh, I should have been more specific. I meant, the newly supported parameter, `maxRowsPerSegment` (or `targetRowsPerSegment` which I suggested below). Neither of them is not described here since we didn't support it before. Maybe https://github.com/apache/druid/blob/master/docs/ingestion/hadoop.md#hash-based-partitioning helps.

Reply:
Added docs for `targetRowsPerSegment`